docs/contributing/999-rfcs/2022-06-14-polycons.md

```diff
@@ -238,7 +238,7 @@ Once the polycon is fully declared, we can define a concrete implementation.
 This is just a `Construct` that implements the desired Polycon's interface and takes the same properties in the constructor. 
 Many of these can exist and each can be composed of any type of construct (from any CDK!). 
 
-The concrete implementation will be chosen automatically through a `PolyconFactory` (see below) when instantiating the polycon.
```

Comment:
We usually don't rewrite history. No need to update old RFCs, especially given it's completely out of date for many other reasons.

Reply:
Oops, looks like this grep went a bit too far - thanks for catching it